spec/features/course_page_spec.rb

```diff
@@ -426,4 +426,61 @@ def js_visit(path, count=3)
       end
     end
   end
+
+  describe 'articles tracking' do
+    let(:course) do
+      editathon_slug = 'This_university.foo.bar/This.course_(term_2015)'
+      create(:editathon,
+             id: 10002,
+             title: 'This.course',
+             slug: editathon_slug,
+             start: '10-07-2019'.to_date,
+             end: '10-08-2019'.to_date,
+             timeline_start: course_start.to_date,
+             timeline_end: course_end.to_date,
+             school: 'This university.foo.bar',
+             expected_students: 1,
+             term: 'term 2015',
+             home_wiki_id: home_wiki.id,
+             description: 'This is a great course',
+             flags: update_logs)
```

Comment:
The more you can rely on the factory defaults, the shorter and easier to understand this test will be. You might be able to just do `create(:editathon)` and have this pass.